src/main/resources/templates/home.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<head th:replace="~{fragments :: header}">
+    <meta charset="UTF-8">
+    <title> Home </title>
+</head>
+<body>
+
+<section id="journal">
+    <h1 id="title">
+        <svg xmlns="http://www.w3.org/2000/svg" width="75" height="75" viewBox="0 0 100 100">
```

Comment:
+1 for svg icons.  (I prefer them in their own files, but I'm not sure if there's a best practice on that.  They'll load faster inline like this.)